### ISSUE TITLE
Can't install oh-my-fish on several (Ubuntu) computers

### ISSUE THREAD

**Issue by tekacs** [oh-my-fish/oh-my-fish#444]
Oh-my-fish failed to install with exactly the below log on two Ubuntu hosts of mine - one is vanilla Ubuntu 16.04 and one is the Linux Subsystem (Ubuntu 14.04) on Windows. They're running fish 2.2.0 and 2.4.0 respectively.

The install runs fine on another host, running Arch with fish 2.4.0.

`~/.config/fish/conf.d/omf.fish` and `~/.local/share/omf` are created successfully, but the errors you see below are being triggered when fish is being launched with those new configs by your install script - if I attempt after installation to launch fish (without removing the above), then I get the same set of errors below.

Any thoughts?

```
fish > curl https://raw.githubusercontent.com/oh-my-fish/oh-my-fish/master/bin/install | fish
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 16530  100 16530    0     0  81264      0 --:--:-- --:--:-- --:--:-- 81428
Installing Oh My Fish to /home/amar/.local/share/omf...
Using release channel "stable".
Cloning master from https://github.com/oh-my-fish/oh-my-fish.git...
Writing bootstrap to /home/amar/.config/fish/conf.d/omf.fish...
Setting up Oh My Fish configuration...
Illegal command name “{$OMF_CONFIG,$OMF_PATH}/pkg/*/key_bindings.fish”
~/.local/share/omf/init.fish (line 33):                       {$OMF_CONFIG,$OMF_PATH}/pkg/*/key_bindings.fish \
                                                              ^
from sourcing file ~/.local/share/omf/init.fish
	called on line 7 of file ~/.config/fish/conf.d/omf.fish

from sourcing file ~/.config/fish/conf.d/omf.fish
	called on line 212 of file /usr/share/fish/config.fish

from sourcing file /usr/share/fish/config.fish
	called during startup

source: Error while reading file “/home/amar/.local/share/omf/init.fish”
fish: Unknown command 'omf install'
Install aborted: Error installing plugins

Oh My Fish installation failed.

If you think that it's a bug, please open an
issue with the complete installation log here:

http://github.com/oh-my-fish/oh-my-fish/issues
```

**Comment by sagebind** [comment]
I think I recall seeing this before... I would check to see what your line ending settings are for Git. Changing the line endings when the repo is cloned can break some things since fish has weird rules on how line endings are interpreted.

I can look into it further in a few hours.

**Comment by tekacs** [comment]
Thanks! It looks like that was exactly the problem - I did the `git`-based installation using:

```
git clone -c core.autocrlf=false https://github.com/oh-my-fish/oh-my-fish
cd oh-my-fish
bin/install --offline
```

... and everything worked perfectly. Hopefully the above steps will help anyone else who runs across this, too. Thanks again!

**Comment by tekacs** [comment]
Ok a further heads-up - omf's installation process will break if `core.autocrlf` is set to true globally, even if the omf repo has the setting turned off as above.

I'm reopening, as this might be worth addressing?

**Comment by sagebind** [comment]
Fixed for the core framework repo by #458.
